stablehlo/reference/ProcessGrid.h

```diff
@@ -35,29 +35,29 @@ namespace stablehlo {
 struct ProcessId;
 
 /// Represents a result of a `ProcessGrid::rendezvous` where multiple processes
-/// synchronize at a barrier and contribute a Tensor each.
-/// This class is pretty much a map from ProcessId to Tensor, with the
+/// synchronize at a barrier and contribute any number of Tensors each.
```

Comment:
"any number of Tensors" is not entirely accurate since we want all processes to send the same number of Tensors to the rendezvous.

Reply:
```
/// Represents a result of a `ProcessGrid::rendezvous` where multiple processes
/// synchronize at a barrier and contribute same number of Tensors.
/// This class is pretty much a map from ProcessId to set of Tensors, with the
/// map-like API.

```
wdyt?